test/FixedGovLst.t.sol

```diff
@@ -177,6 +177,44 @@ contract Constructor is FixedGovLstTest {
     assertEq(fixedLst.SHARE_SCALE_FACTOR(), lst.SHARE_SCALE_FACTOR());
     assertEq(fixedLst.name(), tokenName);
     assertEq(fixedLst.symbol(), tokenSymbol);
+    assertEq(fixedLst.decimals(), 18);
+    assertEq(fixedLst.version(), "2");
+  }
+
+  function test_DOMAIN_SEPARATOR() public view {
+    bytes32 _expectedDomainSeparator = keccak256(
+      abi.encode(
+        // keccak256('EIP712Domain(string name,string version,uint256 chainId,address verifyingContract)')
+        0x8b73c3c69bb8fe3d512ecc4cf759cc79239f7b179b0ffacaa9a75d522b39400f,
+        keccak256(bytes(tokenName)),
+        keccak256(bytes("2")),
```

Comment:
We can read the version string right off the contract instead of hardcoding here.

Reply:
Done: [4fe3dcc](https://github.com/withtally/stGOV/pull/87/commits/4fe3dcc42b5d0c799a165fe42b78f25d23cc1ba5)